flixel/FlxSprite.hx

```diff
@@ -143,6 +143,12 @@ class FlxSprite extends FlxObject
 	 */
 	public var color(default, set):FlxColor = 0xffffff;
 	
+	/**
+	 * Brightens the whole sprite by adding a color (0xRRGGBB format). You can use
+	 * 0xAARRGGBB colors, but the alpha value will simply be ignored. To change the opacity use alpha. 
+	 */
+	public var colorOffset(default, set):FlxColor = 0x000000; 
```

Comment:
Didn't we talk about how FlxColor doesn't really make sense as a representation of color offsets? (negative offsets and all that)